pandas/tests/io/test_compression.py

```diff
@@ -125,3 +127,29 @@ def test_compression_warning(compression_only):
         with tm.assert_produces_warning(RuntimeWarning, check_stacklevel=False):
             with f:
                 df.to_csv(f, compression=compression_only)
+
+
+def test_with_missing_lzma():
+    """Tests if import pandas fails when lzma is not present."""
+    # https://github.com/pandas-dev/pandas/issues/27575
+    code = textwrap.dedent(
+        """\
+        import sys
+        sys.modules['lzma'] = None
+        import pandas
+        """
+    )
+    subprocess.check_output(["python", "-c", code])
+
+
+def test_with_missing_lzma_runtime():
+    """Tests if RuntimeError is hit when calling lzma without
+    having the module available."""
+    code = textwrap.dedent(
```

Comment:
This test needs to be updated a bit. We need

1. `sys.modules['lzma'] = None` to ensure it's not present.
2. I'd like to test an actual writer, rather than this internal function.
3. I'd like to assert something about the output... Maybe do something like

```python
df = pd.DataFrame()
with pytest.raises(RuntimeError, match="lzma module"):
    df.to_csv('foo.csv', compression="xz")
```

Reply:
I will get this done. Thanks for the suggestion.